EIPS/eip-3475.md

```diff
@@ -345,14 +362,17 @@ e.g. In the above example, to get the `symbol` of the given class id, we can use
 
 ## Rationale
 
-**Metadata structure**: Instead of utilizing a mapping from address, the bond's metadata like the time of redemption, redemption conditions, and interest rate can be stored in the bond class and nonce structures. Classes represent the different bond types, and nonces represent the various period of issuances. Nonces under the same class share the same metadata. Meanwhile, nonces are non-fungible. Each nonce can store a different set of metadata. Thus, upon transfer of a bond, all the metadata will be transferred to the new owner of the bond.
+### Metadata structure
+
+Instead of storing the details about the class and their issuances to the user (ie nonce) externally, we store the details in the respective structures. Classes represent the different bond types, and nonces represent the various period of issuances. Nonces under the same class share the same metadata. Meanwhile, nonces are non-fungible. Each nonce can store a different set of metadata. Thus, upon transfer of a bond, all the metadata will be transferred to the new owner of the bond.
 
 ```solidity
  struct Values{
  string stringValue;
  uint uintValue;
  address addressValue;
  bool boolValue;
+ bytes bytesValue;
```

Comment:
This struct isn't part of the Specification section, so it can't really mandate anything. Is it necessary to modify it?